pkg/vm/engine/disttae/engine.go

```diff
@@ -39,32 +40,40 @@ func New(
 var _ engine.Engine = new(Engine)
 
 func (e *Engine) Create(ctx context.Context, name string, op client.TxnOperator) error {
-	//TODO
-	e.newTransaction(op)
-	panic("unimplemented")
+	txn, err := e.getOrAddTransaction(op, e.getClusterDetails)
+	if err != nil {
+		return err
+	}
+	if err := txn.WriteBatch(MO_CATALOG, MO_DATABASE, generateCreateDatabaseTuple(name)); err != nil {
+		return err
+	}
+	return nil
 }
 
 func (e *Engine) Database(ctx context.Context, name string, op client.TxnOperator) (engine.Database, error) {
 	//TODO
-	e.newTransaction(op)
 	panic("unimplemented")
 }
 
 func (e *Engine) Databases(ctx context.Context, op client.TxnOperator) ([]string, error) {
 	//TODO
-	e.newTransaction(op)
 	panic("unimplemented")
 }
 
 func (e *Engine) Delete(ctx context.Context, name string, op client.TxnOperator) error {
-	//TODO
-	e.newTransaction(op)
-	panic("unimplemented")
```

Comment:
A transaction may have mixed insert/delete.     Do you want to separate them or combine them as "write"?

Reply:
My personal preference is to treat them all as write, but this is only a preference, so if people feel that it is better to separate them, it is just as well to do so. @fengttt